ui/components/app/add-network/add-network.stories.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import AddNetwork from './add-network';
 
-const MATIC_TOKEN_IMAGE_URL = './images/matic-token.svg';
+const POL_TOKEN_IMAGE_URL = './images/pol-token.svg';
```

Comment:
i wonder if this was even working before since it should have been `matic-token.png` and that was set to `svg` 🤔

Reply:
It looks like we had both formats previously. I see that the old PNG logo is deleted in this diff, and the SVG is renamed

Reply:
From what I can tell, these were all defined as svg on `develop`

https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/add-network/add-network.stories.js